crates/driver/src/tests/cases/settle.rs

```diff
@@ -111,7 +111,7 @@ async fn test() {
                     }
                 ],
                 "liquidity": [],
-                "effectiveGasPrice": gas_price,
+                "effectiveGasPrice": "0",
```

Comment:
Why? You're introducing yet another unrealistic condition into the test. In the real world, the gas price is not zero.

Reply:
(also re Martin)
As I linked in the description, the correct fix is absolutely to fetch the gas price in the driver. When we do that then the test will fail and we can put back the realistic value here. I didn't do this here because I am fixing multiple things and I felt fetching the real gas price was going to be more work that should go into a separate PR.